dashboard/test/ui/features/pd/teacherApplicationDashboard.feature

```diff
@@ -1,23 +1,35 @@
 @dashboard_db_access
-@eyes
+#@eyes
```

Comment:
I think this should be reënabled

Reply:
Thanks for catching. Leftover from testing